epfl-si/wp-ops
# [feature] Jenkins now reverts to `master` branch before building wp-base

Test procedure:

1. Open [`bc/wp-base`](https://pub-os-exopge.epfl.ch/console/project/wwp-test/edit/builds/wp-base)
2. Click “advanced options” or “Show advanced options”
3. Set “Git Reference” to `tototutu`
4. Redo steps 1–3 in the other builds, i.e. [`bc/httpd`](https://pub-os-exopge.epfl.ch/console/project/wwp-test/edit/builds/httpd), [`bc/httpd`](https://pub-os-exopge.epfl.ch/console/project/wwp-test/edit/builds/httpd), [`bc/cronjob`](https://pub-os-exopge.epfl.ch/console/project/wwp-test/edit/builds/cronjob),  [`bc/wp-ansible-runner`](https://pub-os-exopge.epfl.ch/console/project/wwp-test/edit/builds/wp-ansible-runner)
5. Open [the build job in Jenkins](https://jenkins-wwp-test.epfl.ch/job/wwp-test/job/wwp-test-httpd-jenkins-dev/)
6. Start a new build and wait for it to finish
7. From OpenShift's [build history page](https://pub-os-exopge.epfl.ch/console/project/wwp-test/browse/builds/wp-base?tab=history), check that the last build is a few minutes old and that it completed successfully
8. Browse the corresponding build logs and ensure that the commit ID is that of the current `master` branch
9. Redo steps 7–8 for the other derivative builds, i.e. [`bc/httpd`](https://pub-os-exopge.epfl.ch/console/project/wwp-test/browse/builds/httpd?tab=history), [`bc/mgmt`](https://pub-os-exopge.epfl.ch/console/project/wwp-test/browse/builds/mgmt?tab=history), [`bc/cronjob`](https://pub-os-exopge.epfl.ch/console/project/wwp-test/browse/builds/cronjob?tab=history), [`bc/wp-ansible-runner`](https://pub-os-exopge.epfl.ch/console/project/wwp-test/browse/builds/wp-ansible-runner?tab=history)
